indico/modules/events/timetable/client/js/EntryPopup.tsx

```diff
@@ -94,6 +94,45 @@ function TimetablePopupContent({
     dispatch(actions.setDraftEntry(draftEntry));
   };
 
+  const onDelete = async (e: MouseEvent) => {
+    if (!draftEntry.id) {
+      return;
+    }
+
+    e.stopPropagation();
+
+    const deleteURL = {
+      [EntryType.Break]: breakURL({event_id: eventId, break_id: draftEntry.id}),
+      [EntryType.SessionBlock]: sessionBlockURL({
+        event_id: eventId,
+        session_block_id: draftEntry.id,
+      }),
+      [EntryType.Contribution]: contributionURL({event_id: eventId, contrib_id: draftEntry.id}),
+    }[draftEntry.type];
+
+    try {
```

Comment:
Only wrap the call to `deleteHandler()` in the try block, since we do not want anything that doesn't actually send network requests in there - otherwise any unrelated error would also be caught (and `handleAxiosError` can't really deal with that).